### ISSUE TITLE
Add traitUse()

### ISSUE THREAD

**Issue by ajthinking** [ajthinking/archetype#76]
The ability to get/set which traits a class uses is currently missing.

Consider:
```php
class X
{
    use TraitA, TraitB, TraitC;
    use TraitD;
    use TraitE;
}
```

The variations of how these are grouped poses some questions. What/How should the following return/manipulate?
```php
    PHPFile::load(X::class)->useTrait();
    PHPFile::load(X::class)->add()->useTrait(F::class);
    PHPFile::load(X::class)->set()->useTrait([New1::class, New2::class]);    
```

**Comment by ajthinking** [comment]
Hi @LukasFreyCZ, do you have any preference on this? How would you like it to behave?

**Comment by lukas-frey** [comment]
> 

Hi, thank you for looking at it!

I imagine the `set` function would overwrite all other traits (since I think this is how all the other endpoints work, too).
Each call to `useTrait()` chained with an `add()` call would then append a new line. If you pass an array, it would add multiple traits in one line, if you pass a single class name, it would create a separate trait.

To add the traits from your example, you would run the following:

```php
PHPFile::load(X::class)->add()
->useTrait([TraitA::class, TraitB::class, TraitC::class])
// OR just ->useTrait(TraitA::class, TraitB::class, TraitC::class) using the splat operator.
->useTrait(TraitD::class)
->useTrait(TraitE::class);
```

I personally don't mind too much how the resulting file will look visually in the end, whether it will be one big line with all traits or multiple lines. 

If you need some assistance with the code or testing, I'm available in the evening hours (+20:00 CEST) :)

**Comment by ajthinking** [comment]
Greatly appreciated @LukasFreyCZ , please see #77 for a first iteration of this. I'll put you as reviewer.
EDIT: could not add you as a reviewer for some reason, but anyways :)

**Comment by ajthinking** [comment]
Merged and released in v1.1.5 🎉

**Comment by lukas-frey** [comment]
> Merged and released in v1.1.5 🎉

Thank you so much for adding it. Sorry that I didn't respond sooner, I was out of office.
